index.html

```diff
@@ -144,31 +144,31 @@ <h3>Multi-Post Stories</h3>
                     <button type="button" class="btn">See Project</button>
                 </div>
             </div>
-            </section>
+        </section>
             <section>
-                <div class="contact_us">
+              <div class="contact_us">
                 <h2>CONTACT US</h2>
                 <p class="work_info">
                     If you have an application you are interested in developing,
                     a feature that you need built or a project that needs coding. 
                     I'd love to help with it 
                 </p>
-                <form name="contact_form" class="contact_form">
-                    <input type="text" placeholder="Text Input" class="name">
-                    <input type="email" placeholder="stanleyjones@gmail.com" class="email">
-                    <input type="text" placeholder="Enter text here" class="customer_info">
+                <form action="https://formspree.io/f/daronkwiafe@gmail.com" name="contact_form" class="contact_form" method="post">
+                    <input type="text" placeholder="Name" class="name" required>
```

Comment:
- [ ] It is required to give your name input maximum length of 30 as part of validations  for this input. You can use `maxlength="30"`  attribute to achieve this.